src/main/java/aisu/Aisu.java

```diff
@@ -0,0 +1,93 @@
+package aisu;
+
+import aisu.command.Command;
+import aisu.exception.AisuException;
+import aisu.parser.Parser;
+import aisu.storage.Storage;
+import aisu.task.Task;
+import aisu.tasklist.TaskList;
+import aisu.ui.Ui;
+
+import java.util.List;
+
+
+/**
+ * A tasklist chatbot, named Aisu.
```

Comment:
One empty line is sufficient before JavaDoc
```suggestion
import java.util.List;

/**
 * A tasklist chatbot, named Aisu.
```